packages/ruby/lib/readme/har.rb

```diff
@@ -67,7 +67,7 @@ def response_body
       if @response.content_type == "application/json"
         begin
           parsed_body = JSON.parse(@response.body.first)
-          return filter_with_params(parsed_body)
+          return HarCollection.new(@filter, parsed_body).to_h.to_json
```

Comment:
Do we need an explicit `return` here?

Reply:
We do need this to return early out of the `response_body` method. Otherwise this will always return `@response.body.each.reduce(:+)` on line 77.

Reply:
Right, my suggestion was to convert:

```ruby
if condition
  return value
end

default
````

into the more explicit conditional and avoiding the inline return:

```ruby
if conditional
  value
else
  default
end
```

Reply:
Ah, got it. Will do 👍